open-craft/edx-platform
# [FAL-2358] Update 'View Course' arabic translation

This changes Arabic text on the "View Course" button on the dashboard.

We added the new translation as a suggestion in transifex to hopefully replace the current translation in future releases.

<img width="1337" alt="Screenshot 2021-09-01 at 08 51 43" src="https://user-images.githubusercontent.com/32585/131625950-7f50e3a5-1af6-4764-bdd2-33451b41def5.png">

**Test instructions**:

This patch was installed to [my sandbox](https://mtyaka.opencraft.hosting).

1. Create an account on https://mtyaka.opencraft.hosting
2. Enroll into the demo course
3. Go to [the dashboard](https://mtyaka.opencraft.hosting/dashboard)
4. Verify that the text on the button shows the new translation (استعراض السياق)
